src/main/java/duke/functions/FindTask.java

```diff
@@ -0,0 +1,44 @@
+package duke.functions;
+
+import duke.task.Task;
+
+public class FindTask extends Functions {
+
+    /**
+     * Constructor for an instance of Function.
+     *
+     * @param tl TaskList object that stores all defined tasks
+     * @param st Storage object that controls writing and loading onto/from file
+     */
+    public FindTask(Functions fn) {
+        super(fn.getTl(), fn.getSt(), fn.getOutputLayout());
+    }
+
+    public void find(String query) {
+        boolean flag = false;
+        boolean printed = false;
```

Comment:
Perhaps these booleans could be renamed isFlag and isPrinted to follow naming convention?